agents/bug_reporter/agent.yaml

```diff
@@ -0,0 +1,10 @@
+handle: bug-report
```

Comment:
"Fixie-bug-report" since it's tied to our repo?

Reply:
Well, it's `@fixie/bug-report`. I don't want people having to say `@fixie/fixie-bug-report`, that's silly. The version deployed as the fixie user is scoped to this repo, make sense?